config/initializers/secure_headers.rb

```diff
@@ -72,14 +75,6 @@
     config.csp = default_csp_config
   end
 
-  config.cookies = {
-    secure: true, # mark all cookies as "Secure"
-    httponly: true, # mark all cookies as "HttpOnly"
-    samesite: {
-      lax: true, # SameSite setting.
-    },
-  }
```

Comment:
I think we want to leave this here in the case that `FeatureManagement.rails_csp_tooling_enabled?` is false?